ee/billing/__tests__/usage-reporter.service.spec.ts

```
@@ -98,6 +107,30 @@ describe('UsageReporterService', () => {
     );
   });
 
+  it('should skip organizations without subscriptions', async () => {
+    findManySpy.mockResolvedValue([{ id: 'org-with-sub' }, { id: 'org-without-sub' }]);
```

ESLint

[error] Replace `{·id:·'org-with-sub'·},·{·id:·'org-without-sub'·}` with `⏎······{·id:·'org-with-sub'·},⏎······{·id:·'org-without-sub'·},⏎····`

(prettier/prettier)